From 796d448f7af2488713134d78f5683411d2a734b6 Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Thu, 9 Apr 2015 18:01:51 +0800
Subject: [PATCH] HDMI: call function hdmi->ops->disable() if it is disabled.  
     For system can not be wakeup under following case:       step1. Disable
 HDMI.       step2. set system to deep sleep mode.       step3. remove HDMI
 connection.       step4. wake up system.

Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 drivers/video/rockchip/hdmi/rockchip-hdmi-core.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/video/rockchip/hdmi/rockchip-hdmi-core.c b/drivers/video/rockchip/hdmi/rockchip-hdmi-core.c
index 4fecefed0c65..ee514c3e6d9b 100644
--- a/drivers/video/rockchip/hdmi/rockchip-hdmi-core.c
+++ b/drivers/video/rockchip/hdmi/rockchip-hdmi-core.c
@@ -307,10 +307,8 @@ static void hdmi_work_queue(struct work_struct *work)
 	case HDMI_DISABLE_CTL:
 		if (hdmi->enable) {
 			if (!hdmi->sleep) {
-				/*
 				if (hdmi->ops->disable)
 					hdmi->ops->disable(hdmi);
-				*/
 				hdmi_wq_remove(hdmi);
 			}
 			hdmi->enable = 0;
-- 
2.35.3

